packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation.mdx

```diff
@@ -3,47 +3,75 @@ title: Installation
 description: Install the SDK for video calling
 ---
 
-:::warning
-Write React Native Specifics
+### Prerequisites
 
-This documentation is for the beta version of the Stream Video SDK.
-The SDK is subject to change.
-:::
+First things first, make sure you have set up the development environment for React Native.
+You can find the official guide [here](https://reactnative.dev/docs/environment-setup).
 
-[//]: # ()
-[//]: # (The installation of the packages for the Stream Video React SDK can be either done with `yarn` or `npm`.)
+### Add Stream's Video SDK and its peer dependencies
 
-[//]: # ()
-[//]: # (For `yarn`:)
+In order to install the Stream Video SDK and all its peer dependencies run the following command in your terminal of choice:
 
-[//]: # ()
-[//]: # (```)
+```bash
+yarn add @stream-io/video-react-native-sdk react-native-webrtc react-native-callkeep react-native-device-info react-native-incall-manager react-native-svg text-encoding-polyfill react-native-url-polyfill react-native-get-random-values intl-pluralrules @react-native-community/netinfo @notifee/react-native
+```
 
-[//]: # (yarn add @stream-io/video-react-sdk)
+The following dependencies require additional configuration steps for the installation to be successful and all their functionalities to be available.
+Please follow the links below to complete the installation process:
```

Comment:
I like the thorough explanation and helping users with additional links. They should never feel lost, and this is how to do it. Great job!